packages/next/src/server/app-render/work-unit-async-storage.external.ts

```diff
@@ -38,8 +38,9 @@ export type RequestStore = {
   }
 
   readonly headers: ReadonlyHeaders
-  readonly cookies: ReadonlyRequestCookies
```

Comment:
this seems strange to me at first glance -- ReadonlyRequestCookies type no longer being readonly. I'm sure there's a good reason but probably worth a comment

Reply:
this is for the `synchronizeMutableCookies` thing i added -- i need to reassign it to provide the updated version of the cookies to the render after an action executes.
(we didn't need to do this before because we were incorrectly giving the render `requestStore.mutableCookies`).

i added a similar comment, does that work for you?